XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1593,12 +1589,11 @@
             },
             {
               "Title": "Media Playback",
-              "Uri": "http://windowsstyleguide/controls-and-patterns/media-playback/"
+              "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/media-playback"
             }
           ],
           "RelatedControls": [
-            "Image",
-            "MediaElement"
+            "Image"
```

Comment:
Would AnimatedVisualPlayer also be a related control? (Especially given that AVP doesn't have any related controls at all right now)

Reply:
Maybe? Not really. AVP and MediaPlayerElement have little to nothing in common. Even Image is a stretch, I think.